lib/aggregate/paradigm-assignment.ts

```diff
@@ -215,11 +215,11 @@ export function inferAnalysis({
         throw Error("tie breaker exists but was not applied");
       }
     } else {
-      console.log(`${matches.length} matches for ${head}`);
+      // console.log(`${matches.length} matches for ${head}`);
```

Comment:
Eventually these ambiguous/missing analyses will need to be handled. If you don’t want to print them out, how about counting them and printing a summary at the end?

Reply:
Oops didn't mean to leave that commented out. I was thinking maybe I'd add a `silent` option or yeah, output them to a separate file or something. I think logging a summary is a good idea either way.